subtrack.DAL/Migrations/SubtrackDbContextModelSnapshot.cs

```diff
@@ -15,7 +15,7 @@ partial class SubtrackDbContextModelSnapshot : ModelSnapshot
         protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
-            modelBuilder.HasAnnotation("ProductVersion", "6.0.18");
+            modelBuilder.HasAnnotation("ProductVersion", "7.0.10");
```

Comment:
I don't think this is supposed to be included

Reply:
also autogenerated by the migration

Reply:
> also autogenerated by the migration

Yeah isnt it cuz you are running the app in .net 7?

Reply:
probably, yes

Reply:
fixed it